diff --git a/types/components.d.ts b/types/components.d.ts
index ee6b992b387c319a46c6bd93747292f81b0f984d..fc1764750fe71342c64e2673f77f054ab2366eba 100644
--- a/types/components.d.ts
+++ b/types/components.d.ts
@@ -4,4 +4,5 @@ export { Escape } from './mrkdwn/jsx';
 /** An alias into `JSXSlack.Fragment`, to group a list of JSX elements. */
 export declare const Fragment: import("./jsx-internals").BuiltInComponent<{
     children: JSXSlack.ChildElements;
+    key?: string | number | undefined;
 }>;